### DISCUSSION TITLE
NMEA txt file input

### DISCUSSION THREAD

**Discussion by DMoenning** [jvde-github/AIS-catcher#246] [Q&A] [answered]
Is it possible to feed the catcher with a local txt file with NMEA sentences to view it in the web viewer?

**Comment by jvde-github** [comment, marked as answer]
Yes, the documented approach would be:

AIS-catcher -r txt filename -N 8100

However, this stops the program at the end of the file. I always avoid that with a little trick to read additionally from stdin:

AIS-catcher -r txt filename -r txt . -N 8100

hope that works for you
